internal/serverless/project.go

```diff
@@ -131,6 +138,53 @@ func (p *Project) DefaultFleetServerURL(kibanaClient *kibana.Client) (string, er
 	return fleetURL, nil
 }
 
+func (p *Project) AddLogstashFleetOutput(profile *profile.Profile, kibanaClient *kibana.Client) error {
+	logstashFleetOutput := kibana.FleetOutput{
+		Name:  "logstash-output",
+		ID:    FleetLogstshOutput,
+		Type:  "logstash",
+		Hosts: []string{"logstash:5044"},
```

Comment:
The host is being configured without a protocol. We need to tell the agent to use SSL. Something like this:
```suggestion
		Hosts: []string{"logstash:5044"},
		SSL: &kibana.AgentSSL{
			Enabled: true,
		},
```
Not sure about the exact setting to just enable SSL.

Reply:
The Fleet API [create output](https://www.elastic.co/guide/en/fleet/current/fleet-apis.html#output_create_request) has this [ssl object](https://www.elastic.co/guide/en/fleet/current/fleet-apis.html#output_create_request_elasticsearch_ssl) instead of `ssl_enabled bool`

Tried with just `ca_cert` and no client certificates , But logstash complains with below error

`2024-02-07 15:09:22 io.netty.handler.codec.DecoderException: javax.net.ssl.SSLHandshakeException: Empty client certificate chain`

Hence keeping all of them.